src/mono/wasm/runtime/startup.ts

```diff
@@ -149,6 +153,9 @@ async function preRunAsync(userPreRun: (() => void)[]) {
     await afterInstantiateWasm.promise;
     await afterPreInit.promise;
     if (runtimeHelpers.diagnosticTracing) console.debug("MONO_WASM: preRunAsync");
+    if (MonoWasmThreads) {
```

Comment:
this need be after `mono_wasm_load_config` so that you could get the config for `pthreadPoolSize`

Reply:
maybe I'm confused. Sorry.

Reply:
I verified that if I set `<_WasmPThreadPoolSize>6</_WasmPThreadPoolSize>` in the `browser-threading` sample project, I see this in the console 6 times: `MONO_WASM: replacementAllocateUnusedWorker` and the DevTools debugger shows the right number of workers.  So at least for the non-Blazor path it's working